client/src/components/content/ModelerSidebar/ModelerSidebar.jsx

```diff
@@ -1,25 +1,353 @@
+      currentElement: elementState.currentElement,
+    });
+    updateName(elementState.currentElement.businessObject.name);
+  };
+
+  /**
+   * @description Gets called when a new application was selected in the dropwdown in the sidebar. 
+   * Updates the state of the component and gets the tasks of the application for the TaskDropdown and clears the TaskDropdown.
+   * @param {Object} value new value of the ApplicationDropdown
+   */
+  const applicationChangedHandler = (value) => {
+    elementState.currentElement.businessObject.$attrs[
+      'arkRPA:application'
+    ] = value;
+    setElementState({
+      selectedElements: elementState.selectedElements,
+      currentElement: elementState.currentElement,
+    });
+    if (value) {
+      setRpaApplication(robotId, elementState.currentElement.id, value)
+    }
+    setSelectedApplication(value);
+    resetTask()
+    getTasksForApplication(value);
+  };
+
+  /**
+ * @description TODO
+ */
+  const updateParamSection = (activityId) =>
+    /* this part of code is WIP for implementing the param section
+    const localAttributeObject = JSON.parse(sessionStorage.getItem('attributeLocalStorage'));
+    const localParameterObject = JSON.parse(sessionStorage.getItem('parameterLocalStorage'));
+    const localTaskAplicationCombination = JSON.parse(sessionStorage.getItem('TaskApplicationCombinations'));
+
+    const matchingActivity = localAttributeObject.find((element) => (element.activityId === activityId));
+    const matchingParameterObject = localParameterObject.find((element) => (element.activityId === activityId));
+
+    if (matchingParameterObject) {
+      const matchingCombination = localTaskAplicationCombination.find((element) => (element.Application === matchingActivity.rpaApplication && element.Task === matchingActivity.rpaTask));
+      const copy = matchingParameterObject;
+      copy.rpaParameters.forEach((element) => delete element.value)
+      matchingCombination.inputVars.forEach((element) => copy.rpaParameters.includes(element))
+      return matchingParameterObject;
+    }
+
+
+    // const matchingParameterObject = localParameterObject.find((element) => (element.activityId === activityId && element.matchingActivity.rpaApplication));
+    if (matchingParameterObject) {
+      return matchingParameterObject
+    }
+
+    const inputVars = [];
+    matchingCombination.inputVars.forEach((singleVar) => {
+      const varWithValue = singleVar;
+      varWithValue.value = '';
+      inputVars.push(varWithValue);
+    });
+    const outputVariable = matchingCombination.outputValue ? `${activityId}_output` : null;
+    const newParameterObject = {
+      ssotId: robotId,
+      activityId,
+      outputVariable,
+      rpaParameters: inputVars
+    }
+    localParameterObject.push(newParameterObject);
+    sessionStorage.setItem('parameterLocalStorage', JSON.stringify(localParameterObject));
+    return newParameterObject; */
```

Comment:
If its WIP, can you then please remove it from the pr and maybe stash it/save it on a new branch? There's no need to merge this

Reply:
good idea! I've removed it.